### ISSUE TITLE
Uncaught TypeError: Cannot read properties of null (reading 'classList')

### ISSUE THREAD

**Issue by ArturJo** [heimseiten/contao-timeline-bundle#3]
I get this error in console. 
What do you think. Would it be better to to check if the document is ‘ready’ to be manipulated?

```js
document.addEventListener('DOMContentLoaded', function () {
  // load the timeline js stuff
}, false);
```

![timeline-1](https://user-images.githubusercontent.com/3588345/212071292-592776cf-1ea5-4fad-a34b-177ba4c8f08a.png)

![timeline-2](https://user-images.githubusercontent.com/3588345/212071355-bce99771-d46a-416b-947d-045dd90f2c51.png)

**Comment by heimseiten** [comment]
Should be fixed in version 1.0.14.

**Comment by ArturJo** [comment]
> Should be fixed in version 1.0.14.

It works poperly. Thank you for this quick fix.
